recipes/pycld2/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set name = "pycld2" %}
+{% set version = "0.41" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/pycld2-{{ version }}.tar.gz
+  sha256: a42f6e974df8fdd70685c2baa8a9f523069a260e1140ce604fb9f1fb6c3064df
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
```

Comment:
This looks like it builds C++ files, so I'm pretty sure somewhere you'll need a:
```yaml
    build:
        - compiler('cxx')
```